sdks/python/apache_beam/runners/worker/statesampler.pyx

```diff
@@ -210,26 +251,42 @@ cdef class ScopedState(object):
   cdef readonly StateSampler sampler
   cdef readonly int32_t state_index
   cdef readonly object counter
+  cdef readonly object active_counter
   cdef readonly object name
   cdef readonly int64_t nsecs
+  cdef readonly int64_t active_nsecs
   cdef int32_t old_state_index
 
-  def __init__(self, sampler, name, state_index, counter=None):
+  # Number of completed transitions (entered and exited) since the last sample.
```

Comment:
<!--new_thread; commit:c0086ba69497a5cbd3269f1d691831ca78563dcc; resolved:0-->
Can you give a description of how this is used (either here or in the module docstring)?  Because this doesn't seem to be reported.